schemas/hpc-job-scheduler.md

```diff
@@ -0,0 +1,41 @@
+# HPC Job Scheduler Event Schema
+
+NOTE: Work in Progress!
+
+Tools being targeted:
+
+- Slurm Job Logs
+- Flux Job Logs
+
+## Job Log Schema
+
+| **Field Name**         | **Field Explanation**                                                                | **Slurm**       | **Flux**         |
+| ---------------------- | ------------------------------------------------------------------------------------ | --------------- | ---------------- |
+| job.id                 | Unique identifier of the job                                                         | JobId           | id               |
+| user.name              | User name of who submitted the job                                                   | UserId          | username         |
+| user.id                | User ID number of who submitted the job                                              | UserId          | userId           |
+| group.id               | Group ID number of who submitted the job                                             | GroupId         |                  |
+| group.name             | Group ID number of who submitted the job                                             | GroupId         | jobspec.name     |
+| event.outcome          | The final state of the job                                                           | JobState        | result           |
+| job.queue              | The queue the job was submitted to                                                   | Partition       | queue            |
```

Comment:
completely up to you how you want to format this (i am fine either way), but if we want to prepend the attributes that come from `jobpsec` with `jobspec`, we should maybe do this across all of our fields!

`queue` --> `jobspec.queue`